src/Apps/Artist/artistRoutes.tsx

```diff
@@ -144,12 +144,14 @@ export const artistRoutes: RouteProps[] = [
             $artistID: String!
             $isPrefetched: Boolean!
             $aggregations: [ArtworkAggregation]
+            $input: FilterArtworksInput
```

Comment:
The `prepareVariables` config above on L141 ensures that we [have a proper default](https://github.com/artsy/force/blob/3f74e47f4a92d3fe5f57fdfdd907b5d74e980cf3/src/Apps/Artist/Routes/WorksForSale/Utils/getWorksForSaleRouteVariables.ts#L9) for the `input` argument, but we were not passing it down as needed.